_events/2021/2021-10-intnl-rse-day.md

```diff
@@ -0,0 +1,47 @@
+---
+title: October 2021 International RSE Day Virtual Events
+expires: 2021-10-14
+event_date: "October 14, 2021"
+layout: event
+repeated: false
+---
+
+On October 14th between 11:00 AM and 4:45 PM Eastern time we will be hosting a series of online
+talks and panels with discussion to celebrate the [First International RSE Day](https://researchsoftware.org/council/intl-rse-day.html). Events 
+begin on the hour between 11:00 AM and 4:00 PM Eastern; feel free to join us for some or all
+of the days events.
```

Comment:
"day's" (seem to have lost the suggestion button)

Reply:
thanks - fixed now